### PR TITLE
TPCH Benchmark Scheduler and MVCC 

### PR DESCRIPTION
**Pull request by mrzzzrm**
Introduces command line flags `--mvcc` and `--scheduler` to `hyriseBenchmarkTPCH`

### CODE REVIEW

File: src/benchmark/tpch_benchmark.cpp [hunk 1]
```diff
@@ -321,6 +329,23 @@ int main(int argc, char* argv[]) {
...
+   */
+  if (enable_scheduler) {
+    const auto topology = opossum::Topology::create_numa_topology();
+    opossum::out() << "- Running in multi threaded mode, with the following Topology:" << std::endl;
+    topology->print(opossum::out());
+
+    const auto scheduler = std::make_shared<opossum::NodeQueueScheduler>(topology);
+    opossum::CurrentScheduler::set(scheduler);
+  } else {
+    opossum::out() << "- Running in single threaded mode" << std::endl;
```
Comment (mrks) [inline src/benchmark/tpch_benchmark.cpp]: single-threaded

File: src/benchmark/tpch_benchmark.cpp [hunk 2]
```diff
@@ -321,6 +329,23 @@ int main(int argc, char* argv[]) {
...
 
+  // Display info about MVCC being enabled or not
+  opossum::out() << "- MVCC is " << (enable_mvcc ? "enabled" : "disabled") << std::endl;
+
+  /**
+   * Initialise the Scheduler if the Benchmark was requested to run multithreaded
+   */
+  if (enable_scheduler) {
+    const auto topology = opossum::Topology::create_numa_topology();
+    opossum::out() << "- Running in multi threaded mode, with the following Topology:" << std::endl;
```
Comment (mrks) [inline src/benchmark/tpch_benchmark.cpp]: multi-threaded

File: src/benchmark/tpch_benchmark.cpp (hunk 1 above)
Comment (mrzzzrm) [reply src/benchmark/tpch_benchmark.cpp]: done

File: src/benchmark/tpch_benchmark.cpp (hunk 2 above)
Comment (mrzzzrm) [reply src/benchmark/tpch_benchmark.cpp]: done

**Pull request merged**